Sources/PubNub/Helpers/PAMTokenStore.swift

```diff
@@ -29,22 +29,22 @@ import Foundation
 
 public struct PAMTokenManagementSystem {
   public enum Resource {
-    case user
-    case space
+    case uuid
+    case channel
   }
 
   // Resource stores
-  internal var spaces: [String: PAMToken]
-  internal var users: [String: PAMToken]
+  internal var channels: [String: PAMToken]
+  internal var uuids: [String: PAMToken]
 
   public init(users: [String: PAMToken] = [:], spaces: [String: PAMToken] = [:]) {
-    self.users = users
-    self.spaces = spaces
+    self.uuids = users
```

Comment:
This `init` still seems to reference `users` and `spaces` shouldn't those be renamed?